<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>titleBar</class>
 <widget class="QWidget" name="titleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>492</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="titleLayout">
    <item>
     <widget class="QLabel" name="titleIcon">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="leftBtn">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="titleList"/>
    </item>
    <item>
     <widget class="QPushButton" name="rightBtn">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addNewBtn">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="minimizeBtn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="maximizeBtn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="closeBtn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
